### ISSUE TITLE
How to measure model size?

### ISSUE THREAD

**Issue by yzhang93** [mit-han-lab/haq-release#6]
Thanks for sharing the amazing work! I'm wondering how did you measure the model size. I downloaded your pretrained model 'resnet50_0.1_75.48.pth.tar', and found the size of the model is 102.7MB which is big.

**Comment by kuan-wang** [comment]
We calculate the model size by sum up the product of the bit number and the parameter number of 
 each layer. You can get the centroids and labels refer to https://github.com/mit-han-lab/haq-release/blob/70c2b77b60b1a9a02cba9fc56119ae7b2e1da576/finetune.py#L308
Then you save it in your own format.
